File: core-java-practice/gcr-codebase/programming-elements/Powercal.java
import java.util.Scanner;
public class Powercal {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.println("enter number");
        int num = sc.nextInt();

        System.out.println("enter number power");
        long ans = 1;
        sc.close();
        for(int i = 0;i<num;i++){
             ans *= num;
        }

        System.out.println(ans);
    }
}

File: core-csharp-practice/gcr-codebase/programming-elements/Powercal.cs
using System;
public class Powercal {
    public static void Main(String[] args) {
        int num = 4;
		int powvalue = 2;
		int ans = 1;
        for(int i = 0;i<powvalue;i++){
             ans *= num;
        }

        Console.WriteLine(ans);
    }
}
